src/site/navigation/facility_sidebar_nav.drupal.liquid

```diff
@@ -0,0 +1,40 @@
+            <nav id="va-detailpage-sidebar" class="usa-width-one-fourth va-sidebarnav">
+    <div>
+        <button class="va-btn-close-icon va-sidebarnav-close" type="button" aria-label="Close this menu"></button>
+        <div class="left-side-nav-title">
+            <h4>{{ facilitySidebar.name }}</h4>
+        </div>
+        <ul class="usa-accordion">
+            {% for link in facilitySidebar.links %}
+                <li>
+                    <button class="usa-accordion-button" aria-expanded="false" aria-controls="a{{ forloop.index }}">{{ link.label }}</button>
+                    <div id="a{{ forloop.index }}" class="usa-accordion-content" aria-hidden="false">
+                        {% assign listSize = link.links | size %}
+                        {% if listSize > 1 %}
```

Comment:
How come this is `> 1`? Doesn't this skip lists that contain only one element?

Reply:
Before iterating, sometimes we check for `empty` value, https://github.com/department-of-veterans-affairs/vets-website/pull/9675/files#diff-b64886721b117005d018839ead57b76cR16, if that's any help.